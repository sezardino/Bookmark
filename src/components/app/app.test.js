import React from 'react';
import {shallow} from 'enzyme';

import App from './app';
describe('Component App', () => {
	const app = shallow(<App />);

	it('Component App rendered correctly', () => {
		expect(app.debug()).toMatchSnapshot();
	});
});
