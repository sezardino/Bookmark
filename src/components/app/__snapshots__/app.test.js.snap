// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component App Component App rendered correctly 1`] = `
"<Fragment>
  <Header />
  <Switch>
    <Route path=\\"/\\" exact={true} render={[Function: render]} />
    <Route path=\\"/login\\" render={[Function: render]} />
    <Route path=\\"/pricing\\" render={[Function: render]} />
    <Route path=\\"/contact\\" render={[Function: render]} />
    <Route path=\\"/info\\" render={[Function: render]} />
    <Route path=\\"/extension\\" render={[Function: render]} />
    <Route component={[Function: NoPage]} />
  </Switch>
  <Footer />
  <Popup />
</Fragment>"
`;
